src/app/lists/combo/combo-binding/combo-binding.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="combo-section">
+    <button igxButton="raised" (click)="selectTopItems(true)">Select Most Populated</button>
+    <igx-combo #withValueKey [data]="towns" valueKey="id" displayKey="name" groupKey="country"
+        [(ngModel)]="selectedValueKey">
+    </igx-combo>
+    <div>
+        <label for="codes">Selected IDs:</label>
```

Comment:
Same goes here, actually.. maybe even an actual title instead so we can skip the font style too?